src/pivotal-ui/components/tabs/tabs.scss

```diff
@@ -580,4 +580,7 @@ parent: tabs_react
   </UI.Tab>
 </UI.SimpleAltTabs>
 ```
+
+If you want to customize the tabs, you can use `className` to add a modifier class to the `tab-simple`. `id` and `style` will also be applied to the `tab-simple`.
```

Comment:
We don't need to document this: the prop propagation on `SimpleTabs` and `SimpleAltTabs` is the default behavior and doesn't need to be documented.

The propagation for `Tab` is non-standard and needs documentation.